### ISSUE TITLE
Expose loadEarlierMessagesTillDate

### ISSUE THREAD

**Issue by christianfds** [open-wa/wa-automate-nodejs#2104]
It would be pretty useful to have the **WAPI** method **loadEarlierMessagesTillDate** exposed on the client. 
enabling the obtainment of the messages from a given chat in the past X minutes.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.20.7](https://github.com/open-wa/wa-automate-nodejs/compare/4.20.6...4.20.7) (2021-09-10)

- build(deps): bump axios from 0.21.1 to 0.21.4 [`#2099`](https://github.com/open-wa/wa-automate-nodejs/pull/2099)
- build(deps-dev): bump @typescript-eslint/parser from 4.30.0 to 4.31.0 [`#2110`](https://github.com/open-wa/wa-automate-nodejs/pull/2110)
- feat(Client): expose the loadEarlierMessagesTillDate method [`#2105`](https://github.com/open-wa/wa-automate-nodejs/pull/2105)
- build(deps-dev): bump @types/node from 16.7.8 to 16.9.1 [`#2108`](https://github.com/open-wa/wa-automate-nodejs/pull/2108)
- 🥚 patches update. fixes #2090 [`#2090`](https://github.com/open-wa/wa-automate-nodejs/issues/2090)
- Release 4.20.7 [`2561724`](https://github.com/open-wa/wa-automate-nodejs/commit/2561724419499b1b3c8c2dc192c48bf8a03073a2)
- 🏷️  fix return types for `loadAllEarlierMessages` & `loadEarlierMessagesTillDate` #2104 [`7e792ee`](https://github.com/open-wa/wa-automate-nodejs/commit/7e792ee9899e1e35446c9fe7d0c8b98ab2cad4de)
- 🥚 patches update [`afaee37`](https://github.com/open-wa/wa-automate-nodejs/commit/afaee3775ee7ed400531b022cbaf7a974cc30066)
- 🥚 patches update [`6a33581`](https://github.com/open-wa/wa-automate-nodejs/commit/6a33581f5741dbb0f5d1cfec591b552e9f9ea142)
- 🥚 patches update [`c84c4eb`](https://github.com/open-wa/wa-automate-nodejs/commit/c84c4eb8c7e638e44c2717ad46c548c641a08215)
- 🥚 patches update [`ae9b387`](https://github.com/open-wa/wa-automate-nodejs/commit/ae9b3876338fa5fe160650e616db7fe2a96de552)
- 🥚 patches update [`3883660`](https://github.com/open-wa/wa-automate-nodejs/commit/3883660c33d05abc1f0704d8b10784d7569f96f1)
- updated types-only package [`cea4e72`](https://github.com/open-wa/wa-automate-nodejs/commit/cea4e7215a9c7788b1c524149862f089b21c2cff)
